Prysmctl: Interactive UI is broken for selecting validators to exit

### Describe the bug

The interactive UI for selecting which validator to exit in prysmctl is not rendering correctly. The user is not able to see which validator is currently selected, and long lists are cut off.

I have received a few user reports of this issue and have verified myself. This issue presented itself in `v5.0.0` and is still present in the latest release `v5.0.3` (it works as expected in `v4.2.1`).

### Has this worked before in a previous version?

```text
This worked in all versions `< v5`, verified working in `v4.2.1`.
```


### 🔬 Minimal Reproduction

This issue is very easy to reproduce with the standard command:

```sh
./prysmctl-v5.0.3-linux-amd64 validator exit --wallet-dir=/path/to/validatorwallet
```

Accept the TOS and then try to navigate the list of validators.

### Error

```text
Bugged interactive UI.
```


### Platform(s)

Linux (x86)

### What version of Prysm are you running? (Which release)

prysmctl v5.0.3

### Anything else relevant (validator index / public key)?

_No response_

Hi James, I think it may be that you need a long list of validator keys for the issue to present itself. In order to reproduce I am:

1. Connected to a linux server terminal via SSH
2. Running the `validator exit` command on a validator with many keys (more than one page is key I think for the issue to present itself.

I also triggered a panic using the latest v5.0.3 prysmctl to do a validator exit by jumping around in the (broken) list and then exiting with ctrl-c. **The stack trace is below**, along with images of the issue:

## Here is the latest/broken version (v5.0.3) with panic stack trace

> You can see that the rendering goes beyond the visible range of the terminal. Jumping around in the list causes sporadic rendering and it appears to jump around randomly in the list.

This is rendered first:
![broken-v5 0 3 (1)](https://github.com/prysmaticlabs/prysm/assets/3180446/9dea8807-7e34-45d6-ba09-bbb76f08d431)

Then if navigate with the arrow keys it jumps to the head of the list and you can no longer see the selection:
![broken-v5 0 3 (2)](https://github.com/prysmaticlabs/prysm/assets/3180446/a21710cd-031a-4a8f-bbfb-14b325856cff)

When in this state, I was able to produce a panic by pressing the arrow keys several times and then exiting with CTRL-C:

```log
panic: runtime error: slice bounds out of range [4468:4451]

goroutine 269 [running]:
bytes.(*Buffer).Write(0x0?, {0x2f77324?, 0x0?, 0x0?})
        GOROOT/src/bytes/buffer.go:181 +0xc5
github.com/manifoldco/promptui/screenbuf.(*ScreenBuf).Clear(0xc00033e420)
        external/com_github_manifoldco_promptui/screenbuf/screenbuf.go:47 +0x7c
github.com/manifoldco/promptui/screenbuf.(*ScreenBuf).Write(0xc00033e420, {0xc000c8cd00, 0xe, 0x10})
        external/com_github_manifoldco_promptui/screenbuf/screenbuf.go:68 +0x65
github.com/manifoldco/promptui.(*Select).innerRun.func1({0x31ce6c0?, 0xc00071f080?, 0x2cc9500?}, 0xc00041f030?, 0xba5c48?)
        external/com_github_manifoldco_promptui/select.go:336 +0xa89
github.com/chzyer/readline.(*DumpListener).OnChange(0xc00041efa0?, {0x31ce6c0?, 0x0?, 0xc3a040?}, 0xc000ba5f00?, 0x1?)
        external/com_github_chzyer_readline/operation.go:522 +0x2b
github.com/chzyer/readline.(*Operation).ioloop(0xc0004820e0)
        external/com_github_chzyer_readline/operation.go:341 +0x1a13
created by github.com/chzyer/readline.NewOperation in goroutine 1
        external/com_github_chzyer_readline/operation.go:88 +0x2d6
```

## Here is the prior version which works as expected (v4.2.1)

> You can see that it limits its rendering to a few lines that can be properly navigated with arrow keys

![working-v4 2 1](https://github.com/prysmaticlabs/prysm/assets/3180446/cc70b4d6-6803-4941-bfc6-bae644659672)

--- a/validator/accounts/accounts_helper.go
+++ b/validator/accounts/accounts_helper.go
@@ -49,7 +49,6 @@ func selectAccounts(selectionPrompt string, pubKeys [][fieldparams.BLSPubkeyLeng
 		p := promptui.Select{
 			Label:        selectionPrompt,
 			HideSelected: true,
-			Size:         len(pubKeyStrings),
 			Items:        append([]string{exit, allAccountsText}, pubKeyStrings...),
 			Templates:    templates,
 		}

--- a/validator/accounts/accounts_helper.go
+++ b/validator/accounts/accounts_helper.go
@@ -43,12 +43,14 @@ func selectAccounts(selectionPrompt string, pubKeys [][fieldparams.BLSPubkeyLeng
 	results := make([]int, 0)
 	au := aurora.NewAurora(true)
 	if len(pubKeyStrings) > 5 {
-		log.Warnf("there are more than %d potential public keys to exit, please consider using the --%s or --%s flags", 5, flags.VoluntaryExitPublicKeysFlag.Name, flags.ExitAllFlag.Name)
+		log.Warnf("There are more than %d potential public keys to exit, please consider using the --%s or --%s flags", 5, flags.VoluntaryExitPublicKeysFlag.Name, flags.ExitAllFlag.Name)
 	}
+	log.Infof("Found a total of %d keys", len(pubKeyStrings))
 	for result != exit {
 		p := promptui.Select{
 			Label:        selectionPrompt,
 			HideSelected: true,
+			Size:         10, // Display 10 items at a time.
 			Items:        append([]string{exit, allAccountsText}, pubKeyStrings...),
 			Templates:    templates,
 		}

